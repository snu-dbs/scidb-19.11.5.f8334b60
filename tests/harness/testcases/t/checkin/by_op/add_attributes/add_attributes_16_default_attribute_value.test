--setup
--start-query-logging
store(build(<z:uint64>[i=0:9:0:2],i),a)

--test
scan(a)
add_attributes(a, <y:double default 3.14159>)
scan(a)
store(a, b)
scan(b)
add_attributes(b, <x:uint64 default 256>)
scan(b)
scan(a@1)
scan(a@2)
scan(b@1)
scan(b@2)

--cleanup
remove(a)
remove(b)
--stop-query-logging
