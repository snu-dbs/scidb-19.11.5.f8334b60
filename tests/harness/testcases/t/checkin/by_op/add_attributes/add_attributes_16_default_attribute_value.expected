SCIDB QUERY : <store(build(<z:uint64>[i=0:9:0:2],i),a)>
Query was executed successfully

SCIDB QUERY : <scan(a)>
{i} z
{0} 0
{1} 1
{2} 2
{3} 3
{4} 4
{5} 5
{6} 6
{7} 7
{8} 8
{9} 9

SCIDB QUERY : <add_attributes(a, <y:double default 3.14159>)>
Query was executed successfully

SCIDB QUERY : <scan(a)>
{i} z,y
{0} 0,3.14159
{1} 1,3.14159
{2} 2,3.14159
{3} 3,3.14159
{4} 4,3.14159
{5} 5,3.14159
{6} 6,3.14159
{7} 7,3.14159
{8} 8,3.14159
{9} 9,3.14159

SCIDB QUERY : <store(a, b)>
Query was executed successfully

SCIDB QUERY : <scan(b)>
{i} z,y
{0} 0,3.14159
{1} 1,3.14159
{2} 2,3.14159
{3} 3,3.14159
{4} 4,3.14159
{5} 5,3.14159
{6} 6,3.14159
{7} 7,3.14159
{8} 8,3.14159
{9} 9,3.14159

SCIDB QUERY : <add_attributes(b, <x:uint64 default 256>)>
Query was executed successfully

SCIDB QUERY : <scan(b)>
{i} z,y,x
{0} 0,3.14159,256
{1} 1,3.14159,256
{2} 2,3.14159,256
{3} 3,3.14159,256
{4} 4,3.14159,256
{5} 5,3.14159,256
{6} 6,3.14159,256
{7} 7,3.14159,256
{8} 8,3.14159,256
{9} 9,3.14159,256

SCIDB QUERY : <scan(a@1)>
{i} z
{0} 0
{1} 1
{2} 2
{3} 3
{4} 4
{5} 5
{6} 6
{7} 7
{8} 8
{9} 9

SCIDB QUERY : <scan(a@2)>
{i} z,y
{0} 0,3.14159
{1} 1,3.14159
{2} 2,3.14159
{3} 3,3.14159
{4} 4,3.14159
{5} 5,3.14159
{6} 6,3.14159
{7} 7,3.14159
{8} 8,3.14159
{9} 9,3.14159

SCIDB QUERY : <scan(b@1)>
{i} z,y
{0} 0,3.14159
{1} 1,3.14159
{2} 2,3.14159
{3} 3,3.14159
{4} 4,3.14159
{5} 5,3.14159
{6} 6,3.14159
{7} 7,3.14159
{8} 8,3.14159
{9} 9,3.14159

SCIDB QUERY : <scan(b@2)>
{i} z,y,x
{0} 0,3.14159,256
{1} 1,3.14159,256
{2} 2,3.14159,256
{3} 3,3.14159,256
{4} 4,3.14159,256
{5} 5,3.14159,256
{6} 6,3.14159,256
{7} 7,3.14159,256
{8} 8,3.14159,256
{9} 9,3.14159,256

SCIDB QUERY : <remove(a)>
Query was executed successfully

SCIDB QUERY : <remove(b)>
Query was executed successfully

